checks.d/network.py

```diff
@@ -176,6 +176,8 @@ def _check_linux(self, instance):
                 metrics = self._parse_linux_cx_state(lines[2:], self.TCP_STATES['netstat'], 5)
                 for metric, value in metrics.iteritems():
                     self.gauge(metric, value)
+            except Exception:
+                self.log.exception("Error collecting connection stats.")
```

Comment:
Log or raise? Cause we're logging it with `log.exception()`.... If we raise, then we'd be unable to collect stats such as the network stats below which would be collectible due to procfs being a pseudo FS and not require calling any subcommand.

Not sure what you mean here.